catchup/classBasedPeerSelector_test.go

```diff
@@ -481,6 +482,7 @@ func TestClassBasedPeerSelector_integration(t *testing.T) {
 	require.Nil(t, err)
 	require.Equal(t, mockP1WrappedPeer, peerResult)
 	cps.rankPeer(mockP1WrappedPeer, peerRankNoBlockForRound)
+	cps.rankPeer(mockP1WrappedPeer, peerRankNoBlockForRound)
```

Comment:
why was this extra one needed?

Reply:
in the test above there are 3 failures, one success so need to counter this success with one more failure:
```golang
	oldRank, newRank = cps.rankPeer(mockP1WrappedPeer, peerRankNoBlockForRound)
...
	durationRank = cps.peerDownloadDurationToRank(mockP1WrappedPeer, 500)
	oldRank, newRank = cps.rankPeer(mockP1WrappedPeer, durationRank)
```